lib/shoryuken/util.rb

```diff
@@ -4,13 +4,13 @@ def logger
       Shoryuken.logger
     end
 
-    def fire_event(event, reverse = false)
+    def fire_event(event, options = {}, *event_args)
```

Comment:
I think that makes a lot of sense and I'm in support of that. I'll push a change.